chainerx_cc/chainerx/cuda/float16.cuh

```diff
@@ -71,6 +71,7 @@ public:
     __device__ Float16 Exp() const { return Float16{std::exp(static_cast<float>(*this))}; }
     __device__ Float16 Log() const { return Float16{std::log(static_cast<float>(*this))}; }
     __device__ Float16 Log10() const { return Float16{std::log10(static_cast<float>(*this))}; }
+    __device__ Float16 Log2() const { return Float16{std::log2(static_cast<float>(*this))}; }
```

Comment:
`std::log2` seems not supported in CUDA. Could you fix it likes as follows?
```suggestion
    __device__ Float16 Log2() const { return Float16{std::log(static_cast<float>(*this)) / std::log(2.0) }; }
```

Reply:
Sorry, the above my suggestion seems wrong. Could you fix as follows? 🙇 
```cpp
__device__ inline double Log2(double x) { return std::log2(x); }
__device__ inline float Log2(float x) { return std::log2f(x); }
__device__ inline cuda::Float16 Log2(cuda::Float16 x) { return cuda::Float16{std::log2f(static_cast<float>(x))}; }
```

Reply:
Isn't it for `numeric.cuh`?

Reply:
For here,

```
__device__ Float16 Log2() const { return Float16{std::log2f(static_cast<float>(*this))}; }
```